portal/static/js/src/modules/TnthAjax.js

```diff
@@ -1084,6 +1084,24 @@ export default { /*global $ */
             callback({data: data});
         });
     },
+    "assessmentTimeline": function(userId, params, callback) {
+        callback = callback || function() {};
+        if (!userId) {
+            callback({error: i18next.t("User id is required.")});
+            return false;
+        }
+        this.sendRequest("/api/patient/" + userId + "/timeline", "GET", userId, params, function(data) {
+            if (data) {
+                if (!data.error) {
+                    callback(data);
+                } else {
+                    callback({"error": i18next.t("Error occurred retrieving assessment timeline.")});
```

Comment:
>Error occurred retrieving assessment timeline.

If we want to translate this string, could you replace this error message with an existing (more generic) one that's already translated?